SCClassLibrary/Common/GUI/Base/QSlider2D.sc

```diff
@@ -58,6 +58,9 @@ Slider2D : QAbstractStepValue {
 	incrementY { arg factor=1.0; this.invokeMethod( \incrementY, factor.asFloat ); }
 	decrementY { arg factor=1.0; this.invokeMethod( \decrementY, factor.asFloat ); }
 
+	thumbSize { ^this.getProperty(\thumbSize) }
+	thumbSize_ { |sz| this.setProperty(\thumbSize, sz) }
```

Comment:
`Slider:thumbsize_` uses the arg name `pixels`, perhaps we stay consistent with that? 
Would also disambiguate from the `Size` object.

Reply:
Done.